addons/website/static/src/client_actions/website_preview/website_preview.js

```diff
@@ -215,7 +219,7 @@ export class WebsitePreview extends Component {
         // This is needed for the registerThemeHomepageTour tours
         const { editable, viewXmlid } = this.websiteService.currentWebsite.metadata;
         this.container.el.dataset.viewXmlid = viewXmlid;
-        if (!editable) {
+        if (!editable && !this.testMode) {
```

Comment:
It's confusing IMO: "if we are not in test mode, hide the iframe" -> is not it hidden in test mode? :open_mouth: Maybe `&& this.iframefallback` instead ?

Reply:
> one might think "but why would there is no iframe".

~~Someone who does not speak english might~~ :innocent: 

IMO it's precisely the question I hope they would ask themself, then it's easy to find out why: grep the definition of the `iframefallback` variable, which leads to the qweb condition. It's the same with `!this.testMode` but the first thing you think when reading this IMO is "why not in test mode", then grepping testMode is less straightforward to me :shrug: But it is hard nitpicking, I am fine with both. A comment in any case would be better too IMO.